addons/velero/template/base/install.sh

```diff
@@ -0,0 +1,115 @@
+
+function velero_pre_init() {
+    if [ -z "$VELERO_NAMESPACE" ]; then
+        VELERO_NAMESPACE=velero
+    fi
+    if [ -z "$VELERO_LOCAL_BUCKET" ]; then
+        VELERO_LOCAL_BUCKET=velero
+    fi
+}
+
+function velero() {
+    local src="$DIR/addons/velero/$VELERO_VERSION"
+    local dst="$DIR/kustomize/velero"
+
+    cp "$src/kustomization.yaml" "$dst/"
+
+    velero_binary
+
+    velero_install "$src" "$dst"
+
+    velero_patch_restic_privilege "$src" "$dst"
+
+    kubectl apply -k "$dst"
+
+    kubectl label -n default --overwrite service/kubernetes velero.io/exclude-from-backup=true
+}
+
+function velero_join() {
+    velero_binary
+}
+
+function velero_install() {
+    local src="$1"
+    local dst="$2"
+
+    # Pre-apply CRDs since kustomize reorders resources. Grep to strip out sailboat emoji.
+    $src/assets/velero-v${VELERO_VERSION}-linux-amd64/velero install --crds-only | grep -v 'Velero is installed'
+
+    local resticArg="--use-restic"
+    if [ "$VELERO_DISABLE_RESTIC" = "1" ]; then
+        resticArg=""
+    fi
+
+    local bslArgs="--no-default-backup-location"
+    if ! kubernetes_resource_exists "$VELERO_NAMESPACE" backupstoragelocation default; then
+        bslArgs="--provider aws --bucket $VELERO_LOCAL_BUCKET --backup-location-config region=us-east-1,s3Url=${OBJECT_STORE_CLUSTER_HOST},publicUrl=http://${OBJECT_STORE_CLUSTER_IP},s3ForcePathStyle=true"
```

Comment:
`s3ForcePathStyle=true` most probably needs quotes around `true`

Reply:
It seems to end up quoted in kubectl
```
spec:
  config:
    publicUrl: http://10.96.1.229
    region: us-east-1
    s3ForcePathStyle: "true"
    s3Url: http://rook-ceph-rgw-rook-ceph-store.rook-ceph
  objectStorage:
    bucket: velero
  provider: aws
```

Reply:
nice, just wanted to confirm, thanks!